File: Java/TP3-3/src/Estrela.java

public class Estrela {

	private String nome;
	private double temperatura;
	private String cor;

	public Estrela(String nome, double temperatura, String cor) {
		this.nome = nome;
		this.temperatura = temperatura;
		this.cor = cor;
	}

	public void mostrar() {
		System.out.println("Nome: " + nome + ", Temperatura: " + temperatura + ", Cor: " + cor);
	}

	public double getTemperatura() {
		return temperatura;
	}
}

File: Kotlin/TPK3-3/src/Estrela.kt
class Estrela(nome: String, temperatura: Double, cor: String) {
	private val nome: String
	var temperatura: Double = 0.toDouble()
	private var cor: String

	init {
		this.nome = nome
		this.temperatura = temperatura
		this.cor = cor
	}

	fun mostrar() {
		println(("Nome: " + nome
				+ ", Temperatura: " + temperatura
				+ ", Cor: " + cor))
	}
	fun getTemperatura():Double {
		return temperatura;
	}
}